operator/internal/manifests/internal/sizes.go

```diff
@@ -235,14 +235,14 @@ var StackSizeTable = map[lokiv1.LokiStackSizeType]lokiv1.LokiStackSpec{
 			Global: &lokiv1.LimitsTemplateSpec{
 				IngestionLimits: &lokiv1.IngestionLimitSpec{
 					// Defaults from Loki docs
-					IngestionRate:           4,
-					IngestionBurstSize:      6,
-					MaxLabelNameLength:      1024,
-					MaxLabelValueLength:     2048,
-					MaxLabelNamesPerSeries:  30,
-					MaxLineSize:             256000,
-					PerStreamRateLimit:      3,
-					PerStreamRateLimitBurst: 15,
```

Comment:
The old defaults disappeared because of the initial validation case that not both `desired_rate` and `per_stream_rate_limit` can co-exist. As per the above discussion with @DylanGuedes I added the `per_stream_rate_limit` back but with a new value set to `5 MB`. This means our setups aim for a desired rate of `3 MB/s` but have plenty of room for bursty streams. WDYT?